src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -370,7 +370,7 @@
  *         } catch (ParseException e) {}
  *     }
  * }
- * }</pre></blockquote>
```

Comment:
The example here can utilize enhanced `switch` and pattern matching `instanceof`.